code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -2084,4 +2084,171 @@ All effects don't start immediately, but rather get worse over time; the rate is
+					M.dust(0,1,0)
+
+	return ..()
+
+/datum/reagent/consumable/ethanol/catsip
+	name = "Catsip"
+	id = "catsip"
+	description = "a kawaii drink from space-Japan."
+	color ="#ff99ac"
+	boozepwr = 50
+	quality = DRINK_NICE
+	taste_description = "sweet milkies"
+	glass_icon_state = "catsip"
+	glass_name = "Catsip"
+	glass_desc = "Unfortunately has a tendency to induce the perculiar vocal tics of a wapanese mutant in the imbiber."
+
+/datum/reagent/consumable/ethanol/catsip/on_mob_life(mob/living/M)
+	if(prob(8))
+		M.say(pick("Nya.", "N-nya!", "NYA!"), forced = "catsip")
+	return ..()
+
+/datum/reagent/consumable/ethanol/catsip/on_mob_add(mob/living/carbon/human/M)
+	if(!M.dna.species.is_wagging_tail())
+		M.emote("wag")
+	return ..()
+
+/datum/reagent/consumable/ethanol/rubberneck
+	name = "Rubberneck"
+	id = "rubberneck"
+	description = "A quality rubberneck should not contain any gross natural ingredients."
+	color = "#ffe65b"
+	boozepwr = 60
+	quality = DRINK_GOOD
+	taste_description = "artifical fruityness"
+	glass_icon_state = "rubberneck"
+	glass_name = "Rubberneck"
+	glass_desc = "A popular drink amongst those adhering to an all synthetic diet."
+
+/datum/reagent/consumable/ethanol/duplex
+	name = "Duplex"
+	id = "duplex"
+	description = "An insperable combination of two fruity drinks."
+	color = "#50e5cf"
+	boozepwr = 25
+	quality = DRINK_NICE
+	taste_description = "green apples and blue raspberries"
+	glass_icon_state = "duplex"
+	glass_name = "Duplex"
+	glass_desc = "To imbibe one component separately from the other is consider a great faux pas."
+
+/datum/reagent/consumable/ethanol/trappist
+	name = "Trappist Beer"
+	id = "trappist"
+	description = "A strong dark ale brewed by space-monks."
+	color = "#390c00"
+	boozepwr = 40
+	quality = DRINK_VERYGOOD
+	taste_description = "dried plums and malt"
+	glass_icon_state = "trappistglass"
+	glass_name = "Trappist Beer"
+	glass_desc = "boozy Catholicism in a glass."
+
+/datum/reagent/consumable/ethanol/trappist/on_mob_life(mob/living/carbon/M)
+	if(!M.mind.isholy)
+		return ..()
+	M.adjustFireLoss(-2.5, 0)
+	M.jitteriness = max(0, M.jitteriness-1)
+	M.stuttering = max(0, M.stuttering-1)
+	return ..()
+
+/datum/reagent/consumable/ethanol/blazaam
+	name = "Blazaam"
+	id = "blazaam"
+	description = "A strange drink that few people seem to remember existing. Doubles as a Berenstain remover."
+	boozepwr = 70
+	quality = DRINK_FANTASTIC
+	taste_description = "alternate realities"
+	glass_icon_state = "blazaamglass"
+	glass_name = "Blazaam"
+	glass_desc = "The glass seems to be sliding between realities. Doubles as a Berenstain remover"
+	var/stored_teleports = 0
+	var/mandelamessage
+
+/datum/reagent/consumable/ethanol/blazaam/on_mob_life(mob/living/carbon/M)
+	if(M.drunkenness > 40)
+		if(stored_teleports)
+			do_teleport(M, get_turf(M), rand(1,3), channel = TELEPORT_CHANNEL_BLUESPACE)
```

Comment:
lol

Reply:
is this basically a blink? Also lol getting spaced if you wear anything bluespace.

Reply:
This drink is going to be very rare and the teleportation effects only starts if you are drunk enough.

It causes "fits" of teleportation that are hard to powergame since you will get stunned by vomiting after some of the teleports. If you want this to be more friendly to people wearing bluespace gear i can look into changing that.

Reply:
I changed the teleport channel to anomaly to prevent the spacing issue.